client/api/filesystem/v2alpha1/api.proto

```diff
@@ -15,6 +15,9 @@ service Filesystem {
     // This may be used for unlinking a symlink created through CreateSymlink.
     rpc Rmdir(RmdirRequest) returns (RmdirResponse) {}
 
+    // RmdirContents removes the contents of a directory in the host filesystem.
```

Comment:
maybe add a few more words to mention contents are all the files and sub dirs under the dir. The directory itself will not be deleted.

Reply:
added:

```
    // Unlike Rmdir it won't delete the requested path, it'll only delete its contents.
```